docs/tutorials/load-personio-duckdb.md

```diff
@@ -0,0 +1,65 @@
+# Load Personio Data to DuckDB
+
+Welcome! 👋 This tutorial will guide you through loading data from `Personio` into `DuckDB` using `ingestr`, a command-line tool that enables data ingestion between any source and destination using simple flags, no coding required.
+
+Personio is human resources management software that helps businesses streamline HR processes, including recruitment and employee data management. To analyze and report on this HR data effectively, you may often need to load it into an analytics database like DuckDB. This is where `ingestr` simplifies the process.
+
+## Prerequisites
+Before you begin, ensure you have:
+- [ingestr](../getting-started/quickstart.md#Installation)
+- [Personio API credentials](https://dlthub.com/docs/dlt-ecosystem/verified-sources/personio#grab-credentials)
+- DuckDB
+
+## Configuration Steps 
+### Source Configuration - Personio
+
+#### `--source-uri`
+This flag connects to your Personio account. The URI format is:
+
+```bash
+--source-uri 'personio://?client_id=id_123&client_secret=secret_123'
+```
+
+URI parameters:
+- `client_id`: The client ID used for authentication with the Personio API
+- `client_secret`: The client secret used for authentication with the Personio API
+
+#### `--source-table`
+This flag specifies which table to read from:
+```bash
+--source-table 'employees' # Employee information
+```
+
+### Destination Configuration - DuckDB
+#### `--dest-uri`
+
+This flag connects to DuckDB. The URI format is:
+```bash
+duckdb:///<database-file>
+```
+URI parameters:
+- `database-file`: The path to the DuckDB database file
+
+#### `--dest-table`
```

Comment:
instead of making these as separate sections I'd instead give a full command and explain below that.